### ISSUE TITLE
Documentation: "DNS, Hostname, Alias" page duplication

### ISSUE THREAD

**Issue by datakid** [xcat2/xcat-core#6603]
There are two pages which differ by very little. I can't pick which is preferable, it looks like style guide differences.

In https://github.com/xcat2/xcat-core/tree/master/docs/source/QA

https://github.com/xcat2/xcat-core/blob/master/docs/source/QA/makehosts.rst
and
https://github.com/xcat2/xcat-core/blob/master/docs/source/QA/makehosts_qa.rst

Leading to the documentation having:
https://xcat-docs.readthedocs.io/en/stable/QA/makehosts.html
and 
https://xcat-docs.readthedocs.io/en/stable/QA/makehosts_qa.html

**Comment by gurevichmark** [comment]
It appears that `makehosts_qa` is not being referenced in the docs, only `makehosts.rst`
The `makehosts_qa` should be removed.

```
[c910loginx02 xcat-core]$ git grep makehosts.rst
docs/source/QA/index.rst:   makehosts.rst
[c910loginx02 xcat-core]$ git grep makehosts_qa.rst
[c910loginx02 xcat-core]$
```

**Comment by gurevichmark** [comment]
Resolved by #6707
